README.md

```diff
@@ -3,32 +3,37 @@
 aiXplain Pipelines enables python programmers to add AI functions
 to their software.
 
+An aiXplain pipeline is a directed acyclic graph (DAG) of AI functions built using aiXplain's designer UI. An AI function is a data processing step that relies on a machine learning model to execute. An example of an AI function is speech recognition or machine translation. It helps you process your data by calling a series of functions as defined in the DAG, abstracting the orchestration by providing a simple python function call.
+
+aiXplain has a collection of AI models for each AI function. You can explore the collection of our AI models by using the discover feature of our [platform's website](https://platform.aixplain.com/).
+
+## aiXplain Pipeline Designer DAG
+
+The image below shows a sample aiXplain pipeline built for subtitling video files.
```

Comment:
Maybe add a short description of the pipeline:

The pipeline received an English video as input and first extracts its audio, which is then fed to a VAD node, responsible to find the voice activity segments of the audio input. These segments are then transcribed by a Speech Recognition node and then translated into Spanish by a Translation English-Spanish node. Finally, a Speech Synthesis node is added to dub the Spanish translation.